.github/workflows/main.yml

```diff
@@ -286,6 +286,118 @@ jobs:
           make ENABLE_JIT=1 clean && make ENABLE_EXT_A=0 ENABLE_JIT=1 check -j$(nproc)
           make ENABLE_JIT=1 clean && make ENABLE_EXT_F=0 ENABLE_JIT=1 check -j$(nproc)
           make ENABLE_JIT=1 clean && make ENABLE_EXT_C=0 ENABLE_JIT=1 check -j$(nproc)
+          .ci/riscv-toolchain-install.sh && export PATH=$PWD/toolchain/bin:$PATH
+          python3 -m venv venv
+          . venv/bin/activate
+          .ci/riscv-tests.sh
```

Comment:
commit 9e775bacdd83012137a9aa30eca08216642d33d0 drops unstable architecture test on Arm64/Linux. Remove the RISC-V Architecture Tests here.